internal/pkg/redis/client.go

```diff
@@ -135,7 +112,7 @@ func (c Client) Connect() error {
 
 // Publish sends the provided message to appropriate Redis Pub/Sub.
 func (c Client) Publish(message types.MessageEnvelope, topic string) error {
-	if c.publishClient == nil {
+	if c.redisClient == nil {
 		return pkg.NewMissingConfigurationErr("PublishHostInfo", "Unable to create a connection for publishing")
```

Comment:
Should the error message kind of rewording like host info is empty for publishing?

Reply:
Nice Catch!

Reply:
fixed